todo_app/data/trello_items.py

```diff
@@ -0,0 +1,67 @@
+import requests
+import os
+
+url_main_cards = 'https://api.trello.com/1/cards/'
```

Comment:
I think the constant part of the URL should be defined once and reused in each of the different API calls.

Could you add the following function and use it everywhere it's appropriate?

```suggestion
def build_url(endpoint):
    return 'https://api.trello.com/1' + endpoint
```